python/paddle/fluid/layers/nn.py

```diff
@@ -2305,6 +2306,87 @@ def layer_norm(input,
     return helper.append_activation(layer_norm_out)
 
 
+@templatedoc()
+def group_norm(input,
+               scale=True,
+               shift=True,
+               groups=32,
+               epsilon=1e-05,
+               param_attr=None,
+               bias_attr=None,
+               act=None,
+               data_layout='NCHW',
+               name=None):
+    """
+    **Group Normalization Layer**
+
+    Refer to `Group Normalization <https://arxiv.org/abs/1803.08494>`
+
+    Args:
+        input(Variable): The input tensor variable.
+        scale(bool): Scale is a 1-dimensional tensor of size C that is multiply to the output.
+        shift(bool): Shift is a 1-dimensional tensor of size C that is add to the output.
```

Comment:
There is no need to use `scale(bool)` and ` shift(bool)`, can use `param_attr` and `bias_attr` to determine whether to apply scale and bias.  The `param_attr` and `bias_attr` also can be set True or False in other APIs, like fc.